### ISSUE TITLE
Error in AdaptiveDetector for some videos

### ISSUE THREAD

**Issue by MNMaqsood** [Breakthrough/PySceneDetect#289]
**Bug/Issue Description:**
Hey, I am using the library to find scenes in youtube videos. It is working good on the most videos but for some, I facing this issue for `AdaptiveDetector` 
```
Traceback (most recent call last):
  File "scene_detection/scene_detection.py", line 57, in <module>
    scene_list = find_scenes(video_path, output_path=output_path,
  File "scene_detection/scene_detection.py", line 27, in find_scenes
    scene_manager.detect_scenes(video=video_stream)
  File "/home/user/env/lib/python3.8/site-packages/scenedetect/scene_manager.py", line 813, in detect_scenes
    self._post_process(video.position.frame_num)
  File "/home/user/env/lib/python3.8/site-packages/scenedetect/scene_manager.py", line 696, in _post_process
    self._cutting_list += detector.post_process(frame_num)
  File "/home/user/env/lib/python3.8/site-packages/scenedetect/detectors/adaptive_detector.py", line 152, in post_process
    denominator += self.get_content_val(frame_num + offset)
TypeError: unsupported operand type(s) for +: 'float' and 'NoneType'
```
**Required Information:**
```
video_stream = open_video(video_path, backend='pyav')
stats_manager = StatsManager()
 # Construct our SceneManager and pass it our StatsManager.
 scene_manager = SceneManager(stats_manager)

 # Add ContentDetector algorithm (each detector's constructor
 # takes various options, e.g. threshold).
 detector = AdaptiveDetector(adaptive_threshold=3.0, luma_only=True, min_scene_len=15, min_delta_hsv=15.0, window_width=4, video_manager=None)
 scene_manager.add_detector(detector)
 # Perform scene detection.
 scene_manager.detect_scenes(video=video_stream)
 scene_list = scene_manager.get_scene_list()
```

**Expected Behavior:**
A list containing timestamps of scenes

**Computing Environment:**
 - OS: Ubuntu 22.04
 - Python Version: 3.8.13
 - OpenCV Version: 4.1.2.30

**Additional Information:**
Please note that other detectors i.e ContentDetector and ThresholdDetector are working fine on the video.

**Media [Videos/Images/Screenshots]:**
I tested on this youtube video, you can download it using `youtube-dl`
```
youtube-dl https://www.youtube.com/watch\?v\=HBOpO7RgS64
```

**Comment by MNMaqsood** [comment]
The problem was actually because of video codec - it uses `AV1` codec.
I had to convert it using `ffmpeg`
```
ffmpeg -i demo.mkv -vcodec libx264 -vprofile high -crf 28 -c:v h264 demo.mp4
```

**Comment by wjs018** [comment]
For those running across this in the future, this seems like a similar problem to #86, though with a different codec. If that is the case, per the discussion over there, this is likely due to a bug in OpenCV. The problem could be solved through either updating to a newer version of OpenCV or trying a different backend by specifying `--backend pyav`. API reference for backends can be found [here](https://scenedetect.com/projects/Manual/en/latest/api/backends.html).

**Comment by MNMaqsood** [comment]
Actually I tried to use `pyav` backend but didn't workout and yes the problem is due to `opencv` also discussed in [opencv forum](
https://forum.opencv.org/t/error-your-platform-doesnt-suppport-hardware-accelerated-av1-decoding-when-i-use-cv2-videocapture/9605)

**Comment by Breakthrough** [comment]
@MNMaqsood what happened when you used the PyAV backend?  Would you be willing to test a .whl of the upcoming v0.6.1 version?  This may be resolved due to the changes made in #283, so I want to understand if you still get an app crash with that or if there's still work to be done there.

May I also ask why you set the `video_manager` parameter?  This parameter should never be used, so I want to understand how to make it clear this value should not be set (even if setting to `None`).

**Comment by Breakthrough** [comment]
Closing due to lack of activity, will revisit upon any new updates.
